cluster/ci/config/prow/jobs/cluster-launch-installer-e2e.yaml

```diff
@@ -191,6 +191,7 @@ objects:
           export EMAIL=test@ci.openshift.io
           export PASSWORD=$( date +%s | sha256sum | base64 | head -c 32 ; echo )
           export EC2_AMI_OVERRIDE=ami-02323e873420d8eab
+          export EXPIRATION_DATE=$( date -d '4 hours' --iso=minutes )
```

Comment:
Should I add `--utc` to this to make searching easier?

Reply:
probably

Reply:
> probably

Done with 1d4385c -> f64b8ea.